CONTRIBUTING.md

```diff
@@ -72,44 +74,49 @@ NOTE: Default development database is SQLite3. (Optionally setup and use [Postgr
 1. Clone the repo (`git clone https://github.com/codeforpdx/dwellinglybackend.git`)
 2. Install [pyenv](https://github.com/pyenv/pyenv) | This step is optional but recommended.
    - With pyenv installed pipenv will automatically install the correct python version.
+   - NOTE:
+     - If you previously had the project running, but need to upgrade to Python 3.9.6, check out the OS-specific documentation under [Mac OS Troubleshooting](#Mac-OS-Troubleshooting) and [Note for Windows users](#Note-For-Windows-Users)
```

Comment:
This is quite specific. If we're going to add Python upgrade instructions then I'd like for it to be generic. I don't want to have to update docs every time the python version changes. So perhaps change to something like:
```suggestion
     - If you previously had the project running, but need to upgrade to Python, check out the OS-specific documentation under [Mac OS Troubleshooting](#Mac-OS-Troubleshooting) and [Note for Windows users](#Note-For-Windows-Users)
```